.github/workflows/ci.yaml

```diff
@@ -10,7 +10,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        node: [14, 16, 18, 20]
+        node: [14, 16, 18, 20, 21, 22]
```

Comment:
You can remove 21 since its reached end of life on 2024-06-01

But it would be great to get 22 added 👍 

https://github.com/nodejs/Release?tab=readme-ov-file#end-of-life-releases

Reply:
Will do!

Reply:
We can't do that, even though it is EOL'd many projects will continue to run it, and therefore it should be tested.

Reply:
That's up to you. I just noticed that 15, 17, and 19 are missing. So I figured you only tested LTS versions.